pyNN/common/populations.py

```diff
@@ -828,6 +832,7 @@ def __init__(self, parent, selector, label=None):
         self.local_cells = self.all_cells[self._mask_local]
         self.first_id = numpy.min(self.all_cells)  # only works if we assume all_cells is sorted, otherwise could use min()
         self.last_id = numpy.max(self.all_cells)
+        self.annotations = {}
```

Comment:
If we remove annotations from here then it would not be possible to supply annotations to a single cell from a population, such as:
`record(["spikes", "v"], cells[0], filename_sim, annotations={'script_name': __file__})`
It would fail here:
https://github.com/appukuttan-shailesh/PyNN/blob/master/pyNN/common/procedural_api.py#L80